packages/lib/survey/service.ts

```diff
@@ -320,6 +330,37 @@ export const getSurveys = async (environmentId: string, page?: number): Promise<
   return surveys.map((survey) => formatDateFields(survey, ZSurvey));
 };
 
+export const getSurveyCountByEnvironmentId = async (environmentId: string): Promise<number> => {
```

Comment:
We have an implicit convention where you don't need the `ByEnvironmentId` (or similar) if it's the default parent of the object (so that we get simple `getSurveys` or other functions). So in this case `ByEnvironmentId` is not needed.